src/web/basics/form-elements/dropdown/dropdown.js

```diff
@@ -0,0 +1,125 @@
+import './dropdown.scss';
+
+const Template = require('./dropdown.html');
+const Interface = require('interface.json');
+const Core = require('_core.js');
+
+const TextField = require('../text-field/text-field.js');
+const Option = require('./option/option.js');
+
+const OPEN_CLASS = 'hig__dropdown--open';
```

Comment:
this is what I need to do now that these classes are set in place for input buttons